src/Functions/parseDateTime.cpp

```diff
@@ -150,7 +150,7 @@ namespace
             if (text == "bc")
                 throw Exception(ErrorCodes::CANNOT_PARSE_DATETIME, "Era BC exceeds the range of DateTime");
             else if (text != "ad")
-                throw Exception(ErrorCodes::CANNOT_PARSE_DATETIME, "Unknown era {}", text);
+                throw Exception(ErrorCodes::CANNOT_PARSE_DATETIME, "Unknown era {} (expected 'ad' or 'bc')", text);
```

Comment:
"expected 'ad' only"? 

bc is not allowed as per the line above.

Reply:
Well, technically `bc` is expected too :) Yes, it's not supported, but it's a valid value